### PR TITLE
fix replace the global var

### PR DESCRIPTION
**Pull request by Yancey0623**
Omit work of #469

### CODE REVIEW

File: sql/sql_test.go [hunk 1]
```diff
@@ -76,13 +76,13 @@ func TestMain(m *testing.M) {
 	dbms := getEnv("SQLFLOW_TEST_DB", "mysql")
 	switch dbms {
 	case "sqlite3":
-		testDB := testSQLiteSQL()
+		testDB = testSQLiteSQL()
```
Comment (tonyyang-svail) [inline sql/sql_test.go]: shall we also rename `testDB` in `testHiveSQL` to `db`? So that the global `testDB` is not assigned twice.

Comment (wangkuiyi) [reply sql/sql_test.go]: I understand @tonyyang-svail 's comment as to rewrite

```go
testDB, e = Open("hive://root:root@localhost:10000/churn")
```

in function `testHiveSQL` as

```go
db, e := Open("hive://root:root@localhost:10000/churn")
```

I agree with him and likewise we should change the assignments to testDB in other functions -- testSQLLiteSQL and testMySQLSQL.

-----

It seems that we should also rename 

- testHiveSQL into testHiveDatabase
- testSQLLiteSQL into testSQLLiteDatabase
- testMySQLSQL into testMySQLDatabase

Comment (Yancey0623) [reply sql/sql_test.go]: Thx @tonyyang-svail and @wangkuiyi ! Done.

**Review by tonyyang-svail — APPROVED** [review]
LGTM.

**Pull request merged**